app/models/unit.rb

```diff
@@ -1669,11 +1652,12 @@ def tasks_for_task_inbox(user)
   #   task_def_id => { ... }
   #
   def task_status_stats
+    # TODO (stream) Do we need tutorial here?
```

Comment:
@macite any suggestions?

Reply:
Yes -> see the comment. Tutorial is essential here.